### ISSUE TITLE
Data batching/chunking on AMI deployment breaks on log file

### ISSUE THREAD

**Issue by zagorsky** [onnela-lab/beiwe-backend#30]
Error report from Zhe:
> after a quick manual hotfix to unlock the database and run the rest of the code, the chunk processing seems work well, getting the correct patient IDs and number of data files, except it stops when it doesn't recognize the `log` data type. It doesn't skip the file.  error text: `data type unknown: [RANDOM_STRING]/[PATIENT_ID]/ios/log/1519082358137.csv`

Does this error happen only on the AMI, or also on other deployment types?

**Comment by writezhe** [comment]
I believe I isolated the error in the AMI version (given my understanding of the process). 
In `config/constants.py`, for the `UPLOAD_FILE_TYPE_MAPPING` dict, there is no appropriate entry for the iOS app logs. The Android version uses a folder named `logFile`, which is there, but the iOS app stores the log in `ios/log/[number].csv`.

To address this with a quick hack, I added an item to the dict named `ios` with the same `LOG_FILE` value. After this, running `ami_cron_target.py` manually works for me and I can download chunked data from the beiwe portal.

(Thanks for the help and creating these bug reports earlier!)
